Sources/PrivacyDashboard/Model/TrackerInfo.swift

```diff
@@ -21,7 +21,7 @@ import TrackerRadarKit
 import ContentBlocking
 
 public struct TrackerInfo: Encodable {
-    
```

Comment:
all white space changes have been done by `swiftlint --fix`, the code indentation was not consistent across the project. I'll try to ignore the white spaces when pushing

Reply:
please use "Hide white spaces" here, after this fix this shouldn't be a problem anymore
![Uploading Screenshot 2023-11-20 at 12.42.20.jpg…]()